src/gui/operations/SaveOperation.cpp

```diff
@@ -91,7 +91,6 @@ SaveResultDto SaveOperation::save(const QString &path)
 
 void SaveOperation::notify(const QString &title, const QString &message, const QString &path, NotificationTypes notificationType) const
 {
-	auto parentDirectory = PathHelper::extractParentDirectory(path);
-	NotifyOperation operation(mToastService, title, message + QLatin1String(" ") + path, parentDirectory, notificationType);
+	NotifyOperation operation(mToastService, title, message + QLatin1String(" ") + path, path, notificationType);
```

Comment:
Isn't it enought to include the path only one? you can extract the parent path from the path I think.

Reply:
I am only including the `path` once, as the 4th parameter for the ctor.

Reply:
Ah, yes, you're right, just saw two times path.